syntax = "proto3";

package api.user.request;

message GetConnections {
    message Filter {
        optional bool established = 1;
    }
    Filter filter = 1;
}
